### PR TITLE
Jess MultiGallery Feedback

### PR DESCRIPTION
**Pull request by tbezman**
image

### CODE REVIEW

File: src/components/core/IconContainer.tsx [hunk 1]
```diff
@@ -105,14 +107,15 @@ function IconContainer(
     >
       {disableHoverPadding && <HoverCircle variant={variant} size={size} disabled={disabled} />}
 
-      {icon}
+      <HStack justify="center" align="center" style={{ position: 'relative', zIndex: 1 }}>
+        {icon}
+      </HStack>
```
Comment (tbezman) [inline src/components/core/IconContainer.tsx:112]: fixes weird layering

File: src/components/GalleryEditor/getInitialCollectionsFromServer.ts [hunk 2]
```diff
@@ -52,6 +58,34 @@ export function getInitialCollectionsFromServer(
...
+
+      sections: {
+        [generatedSectionId]: initialSection,
+      },
+      activeSectionId: generatedSectionId,
+    };
+
+    collections[generatedCollectionId] = initialCollection;
+  }
+
```
[the remark this answers is not in the document]
Comment (Robinnnnn) [reply src/components/GalleryEditor/getInitialCollectionsFromServer.ts:88]: nit: add comment above this block `// generate default collection client-side if user doesn't have any`

File: src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx [hunk 3]
```diff
@@ -37,6 +38,21 @@ export function OnboardingEditGalleryNavbar({
...
+        event.preventDefault();
+
+        onDone();
+      }
+    }
+
+    window.addEventListener('keydown', handlePress);
+
+    return () => window.removeEventListener('keydown', handlePress);
+  }, [onDone]);
```
Comment (Robinnnnn) [inline src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx:54]: can we use our existing `useMultiKeyDown`?

File: src/contexts/globalLayout/GlobalNavbar/CollectionSaveButtonWithCaption.tsx [hunk 4]
```diff
@@ -65,6 +65,23 @@ export function CollectionSaveButtonWithCaption({
...
+        if (!isPopupDisplayed && !disabled) {
+          handleOpenCaption();
+        }
+      }
+    }
+
+    window.addEventListener('keydown', handlePress);
+
+    return () => window.removeEventListener('keydown', handlePress);
+  }, [disabled, handleOpenCaption, isPopupDisplayed]);
```
Comment (Robinnnnn) [inline src/contexts/globalLayout/GlobalNavbar/CollectionSaveButtonWithCaption.tsx:83]: same here

**Review by Robinnnnn — APPROVED** [review]

File: src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx (hunk 3 above)
Comment (tbezman) [reply src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx:54]: i think we should toss that and rely on something more battle tested. https://react-hotkeys-hook.vercel.app/

Comment (tbezman) [reply src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx:54]: will do this in a separate pr right now

**Pull request merged**